extension/show-names.js

```diff
@@ -0,0 +1,57 @@
+window.showRealNames = () => {
+	const storageKey = 'cachedNames';
+
+	const getCachedUsers = cb => {
+		chrome.storage.local.get(storageKey, data => cb(data[storageKey]));
+	};
+
+	const updateCachedUsers = users => {
+		chrome.storage.local.set({[storageKey]: users});
+	};
+
+	const addUsersName = (user, name) => {
+		const usernameLinks = document.querySelectorAll(`.timeline-comment-header-text:not(.has-full-name) a[href="/${user}"]`);
+		for (const userLink of usernameLinks) {
+			const nameSpan = `<span class="comment-full-name">(${name}) -</span>`;
+			userLink.insertAdjacentHTML('afterend', nameSpan);
+			userLink.closest('.timeline-comment-header-text').classList.add('has-full-name');
+		}
+	};
+
+	getCachedUsers(users => {
+		const usersOnPage = Array.from(document.querySelectorAll('.js-discussion .author')).map(el => el.innerText);
+		const uniqueUsers = new Set(usersOnPage);
+
+		// Add cached users to DOM first, since the calls for everyone else will be slow
+		for (const user of uniqueUsers) {
+			const cachedName = users[user];
+			if (cachedName) {
+				addUsersName(user, cachedName);
+				uniqueUsers.delete(user);
+			}
+		}
+
+		const userUrl = user => `//github.com/${user}`;
```

Comment:
Specify `https` protocol.

See: http://www.paulirish.com/2010/the-protocol-relative-url/